docs/reference/release-notes/highlights-7.4.0.asciidoc

```diff
@@ -0,0 +1,106 @@
+[[release-highlights-7.4.0]]
+== 7.4.0 release highlights
+++++
+<titleabbrev>7.4.0</titleabbrev>
+++++
+
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-highlights[]
+[float]
+==== Results pinning
+
+With the release of Elasticsearch 7.4.0, we provide the ability to define the first records in a result set (and the order in which they will show up) directly within Elasticsearch. This capability is provided by the new https://www.elastic.co/guide/en/elasticsearch/reference/7.4/query-dsl-pinned-query.html[pinned query].
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== New shape field type
+
+Elasticsearch introduces a new https://www.elastic.co/guide/en/elasticsearch/reference/7.4/shape.html[shape field type], which allows users to position (and query) shapes in an arbitrary geometry of their choosing.
```

Comment:
```suggestion
A new <<shape,`shape`>> field type has been added,
which allows you to position and query shapes
in a geometry of your choosing.
```

I don't think we need both 'arbitrary' and 'of your choosing.' However, please re-add if you feel both are needed.